src/test/java/apicurito/tests/utils/slenide/MainPageUtils.java

```diff
@@ -62,14 +65,28 @@ public static SelenideElement getDataTypeWithName(String datatypeName) {
                 .$$(By.className("api-definition")).filter(text(datatypeName)).first();
     }
 
-    public static void createDataType(String name, String description, Boolean isRest) {
+    public static void createDataType(String name, String description, String example, Boolean isRest) {
         log.info("Creating data type with name {} and description {} and rest resources are {}", name, description, isRest);
 
-        CommonUtils.getLabelWithName("name", CommonUtils.getAppRoot()).setValue(name);
+        SelenideElement entityForm = CommonUtils.getAppRoot().$(MainPageElements.ENTITY_EDITOR_FORM);
+
+        CommonUtils.getLabelWithName("name", entityForm).setValue(name);
+
+        if (!description.isEmpty()) {
+            entityForm.$(By.id("description")).$("ace-editor textarea").sendKeys(description);
```

Comment:
Can we add `ace-editor textarea`  selector to `Elements.` inner class?

Reply:
added